### PR TITLE
Fix typo in source code comment

### PR DESCRIPTION
**Pull request by MichaelCurrin**

### CODE REVIEW

File: lib/jekyll-seo-tag/url_helper.rb [hunk 1]
```diff
@@ -8,8 +8,8 @@ module UrlHelper
 
       # Determines if the given string is an absolute URL
       #
-      # Returns true if an absolute URL.
```
Comment (ashmaroli) [inline lib/jekyll-seo-tag/url_helper.rb:11]: Why was this *period* removed, @MichaelCurrin?

Comment (MichaelCurrin) [reply lib/jekyll-seo-tag/url_helper.rb:11]: The two lines below and the one above do not have one. So made it consistent.

Comment (ashmaroli) [reply lib/jekyll-seo-tag/url_helper.rb:11]: Fair enough.

**Review by ashmaroli — APPROVED** [review]

**Comment by ashmaroli** [comment]
Thanks @MichaelCurrin 
@jekyllbot: merge +dev

**Pull request merged**